requirements.txt

```diff
@@ -0,0 +1,2 @@
+ddt
```

Comment:
do we need to specify versions?

Reply:
do we? 
requirements.txt is a common approach for pip dependencies 
not sure we can specify a minimal version, but we can look into it when if/when we figure out what that is